### ISSUE TITLE
in_winevtlog: StringInserts field not present in resulting logs by default

### ISSUE THREAD

**Issue by braydonk** [fluent/fluent-bit#8854]
## Bug Report

**Describe the bug**
`StringInserts` are no longer present in Windows Event Logs from `in_winevtlog` by default like they should be.

**To Reproduce**
Run the latest Fluent Bit. I'm not sure what version exactly it was introduced. 

**Expected behavior**
Should contain `StringInserts`, as `string_inserts` in the config map is supposed to default to `true`.

**Your Environment**

* Version used: 3.0.4
* Configuration:
```
[INPUT]
    Channels            Application,System
    Interval_Sec        1
    Name                winevtlog
    Render_Event_As_XML True
    Tag                 winlog2.winlog2

[OUTPUT]
    Name stdout
    Match *
```
* Environment name and version (e.g. Kubernetes? What version?): Windows Server 2019
* Server type and version: Windows Server 2019
* Operating System and version: Windows Server 2019
* Filters and plugins: in_winevtlog

**Additional context**
I opened a PR with an explanation and a fix here: https://github.com/fluent/fluent-bit/pull/8853  
TL;DR is that a `size_t` is written into an `unsigned int` field in the config struct which overwrites the next field since it's 8 bytes wide on a 64 bit machine.  
Alternatively this could just not use `FLB_CONFIG_MAP_SIZE`, I am unsure if there is a particular reason that this was using `unsigned int`.

**Comment by braydonk** [comment]
The PR is merged, looks like this will make it into Fluent Bit 3.0.5.
